qiskit_ibm_provider/job/ibm_circuit_job.py

```diff
@@ -984,20 +786,25 @@ def _get_qobj(self) -> Optional[Union[QasmQobj, PulseQobj]]:
 
         return self._qobj
 
-    def _extract_client_version(self, data: Dict[str, str]) -> Dict:
+    def _extract_client_version(self, data: str) -> Dict:
         """Extract client version from API.
 
         Args:
             data: API client version.
 
         Returns:
             Extracted client version.
-        """
-        if data:
-            if data.get("name", "").startswith("qiskit"):
-                return dict(zip(data["name"].split(","), data["version"].split(",")))
 
-            return {data.get("name", "unknown"): data.get("version", "unknown")}
+        Additional info:
+            The runtime client returns the version as a string, e.g.
+            "0.1.0,0.21.2,0.10.4"
+            Where the numbers represent versions of qiskit-ibm-provider, qiskit-terra and qiskit-aer
+        """
+        if data is not None:
+            if "," not in data:  # sometimes only the metapackage version is returned
+                return {"qiskit": data}
+            client_components = ["qiskit-ibm-provider", "qiskit-terra", "qiskit-aer"]
```

Comment:
Is this correct? What if the user didn't have qiskit-aer installed?

Reply:
Yes I believe this is correct - for the most part, only the metapackage version is returned. If multiple versions are returned, they should always be returned in the order of provider, terra, aer.